app/src/App.js

```diff
@@ -82,6 +82,12 @@ class App extends Component {
       return Array.prototype.map.call(new Uint8Array(buffer), x => ('00' + x.toString(16)).slice(-2)).join('');
     }
 
+    function isValidAddress(address) {
+      if (address.substr(0, 2) != '0x')
+        address = '0x' + address;
+      return /^0x[0-9a-fA-F]{40}$/.test(address);
```

Comment:
You could also use the same helper from `ethereumjs-util`, not sure it makes sense to use it though since this is already written and we plan to move this into etherchain.

Reply:
I was thinking in using the helper from `ethereumjs-util` at first, but didn't want to import a whole module just to use one function.